drivers/bus/isapnp/isapnp.c

```diff
@@ -1223,9 +1219,7 @@ IsaPnpCreateReadPortDOResources(
         Descriptor++;
     }
 
-    PdoExt->ResourceList = ResourceList;
-    PdoExt->ResourceListSize = ListSize;
```

Comment:
You don't need to return the size of the allocated resource list anymore?

Reply:
The read data port resource list is now generated on the fly whenever we request the list. The `ResourceListSize` field is being used as a data storage for IsaPnP logical devices.

Reply:
Can you add a comment in the source for that `ResourceListSize` then, telling it has nothing to do with the `ResourceList` ? (or use a different name for it, etc.?) :thinking:

Reply:
> The read data port resource list is now generated on the fly whenever we request the list.

This is a wanted behaviour, in order to have the driver/caller always getting up-to-date information?

Reply:
Please check the first commit in the series about this behavior.